EXM_104/Makefile

```
@@ -53,34 +57,70 @@ connectathon:
 	curl -X POST \
   http://localhost:8080/cqf-ruler-dstu3/fhir \
   -H 'Content-Type: application/json' \
-	-d @connectathon/fhir3/resources/library/all-libraries-bundle.json
+	-d @connectathon/fhir3/bundles/EXM104_FHIR3-8.1.000/EXM104_FHIR3-8.1.000-bundle.json
 	curl -X POST \
   http://localhost:8080/cqf-ruler-dstu3/fhir/Library \
   -H 'Content-Type: application/json' \
 	-d @connectathon/fhir3/resources/library/library-EXM104_FHIR3-8.1.000.json
 	curl -X POST \
   http://localhost:8080/cqf-ruler-dstu3/fhir/Library \
   -H 'Content-Type: application/json' \
-	-d @connectathon/fhir3/resources/library/library-TJC_Overall_FHIR3-3.6.000.json
+	-d @connectathon/fhir3/resources/library/library-TJCOverall_FHIR3-3.6.000.json
 		curl -X POST \
   http://localhost:8080/cqf-ruler-dstu3/fhir/Library \
   -H 'Content-Type: application/json' \
 	-d @connectathon/fhir3/resources/library/library-MATGlobalCommonFunctions_FHIR3-4.0.000.json
 	touch .seed-cqf-ruler-no-vs
 
+.seed-cqf-ruler-r4:
+	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-r4`; do printf '.'; sleep 5; done
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-r4/fhir/ \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/fhir4/bundles/EXM104_FHIR4-8.1.000/EXM104_FHIR4-8.1.000-bundle.json
+	touch .seed-cqf-ruler-r4
+
+.seed-cqf-ruler-no-vs-r4:
+	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-r4`; do printf '.'; sleep 5; done
+	curl -X PUT \
+  http://localhost:8080/cqf-ruler-r4/fhir/Measure/measure-exm104-FHIR4 \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/fhir4/resources/measure/measure-EX104_FHIR4-8.1.000.json
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-r4/fhir/ \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/fhir4/bundles/EXM104_FHIR4-8.1.000/EXM104_FHIR4-8.1.000-files/library-EXM104_FHIR4-8.1.000.json
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-r4/fhir \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/fhir4/resources/library/all-libraries-bundle.json
+	touch .seed-cqf-ruler-no-vs-r4
+
 synthea:
-	git clone --single-branch --branch abacus --depth 1 https://github.com/projecttacoma/synthea.git
+	git clone --single-branch --branch exm_104_r4 --depth 1 https://github.com/projecttacoma/synthea.git
```

```suggestion
	git clone --single-branch --branch abacus --depth 1 https://github.com/projecttacoma/synthea.git
```

Will need to be updated once https://github.com/projecttacoma/synthea/pull/10 gets in